Modules/Filtering/FFT/include/itkFFTWCommon.h

```diff
@@ -129,14 +129,12 @@ class Proxy<float>
                ComplexType * in,
                PixelType *   out,
                unsigned int  flags,
-               int           threads = 1,
+               int           itkNotused(threads) = 1,
                bool          canDestroyInput = false)
   {
 #  ifndef ITK_USE_CUFFTW
     const std::lock_guard<FFTWGlobalConfiguration::MutexType> lockGuard(FFTWGlobalConfiguration::GetLockMutex());
     fftwf_plan_with_nthreads(threads);
```

Comment:
Requested changes because the `threads` parameter is actually used here!!! So please do `[[maybe_unused]] threads`, instead of `itkNotused(threads)`!